src/drand/timelock-encrypter.ts

```diff
@@ -14,22 +12,20 @@ export function createTimelockEncrypter(client: ChainClient, roundNumber: number
 
     return async (fileKey: Uint8Array): Promise<Array<Stanza>> => {
         const chainInfo = await client.chain().info()
+        const pk = Buffer.from(chainInfo.public_key, "hex")
         const id = hashedRoundNumber(roundNumber)
-        let ciphertext: Ciphertext<Point>
+        let ciphertext: Ciphertext
         switch (chainInfo.schemeID) {
             case "pedersen-bls-unchained": {
-                const point = PointG1.fromHex(chainInfo.public_key)
-                ciphertext = await ibe.encryptOnG1(point, id, fileKey)
+                ciphertext = await ibe.encryptOnG1(pk, id, fileKey)
             }
                 break;
             case "bls-unchained-on-g1": {
-                const point = PointG2.fromHex(chainInfo.public_key)
-                ciphertext = await ibe.encryptOnG2(point, id, fileKey)
+                ciphertext = await ibe.encryptOnG2(pk, id, fileKey)
             }
                 break;
             case "bls-unchained-g1-rfc9380": {
-                const point = PointG2.fromHex(chainInfo.public_key)
-                ciphertext = await ibe.encryptOnG2RFC9380(point, id, fileKey)
+                ciphertext = await ibe.encryptOnG2RFC9380(pk, id, fileKey)
```

Comment:
This switch statement seems useless now?
We used to set `point` in it, but now seems we just feed it `pk` without need for a difference between cases